openlibrary/macros/AffiliateLinks.html

```diff
@@ -0,0 +1,34 @@
+$def with (opts)
+
+$ isbn = opts['isbn']
+$ asin = opts['asin']
+$ scribd = opts['scribd']
```

Comment:
Is the `opts` which gets passed into the templator always guaranteed to have these three keys: isbn, asin, scribd? What happens if the key is not present? Would it make sense to do something like opts.get('isbn', '')?